src/config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
+
+const port = '8080';
+const host = '';
+
+const config = () => {
+    return {
+        port: port? port : process.env.PORT,
```

Comment:
Maybe I'm wrong, but aren't `port` and `host` declared just above this code? This way we'll never get values from the environment variables.

Reply:
Well, shouldn't we declare port and host there? It's config.js after all...